docs/DeveloperGuide.md

```diff
@@ -944,26 +971,110 @@ testers are expected to do more *exploratory* testing.
     1. Prerequisites: Viewing only some contacts using the `find` or `findAny` command. Multiple contacts in the list.
     
     1. Test Case: `delete 1`<br>
-       Expected: First contact that is being viewed is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+       Expected: First contact that is being viewed is deleted from the list. Details of the deleted contact shown in the status message.
 
    1. Test case: `delete 0`<br>
-      Expected: No contact is deleted. Error details shown in the status message. Status bar remains the same.
+      Expected: No contact is deleted. Error details shown in the status message.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the size of the contacts that are currently being viewed)<br>
+   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the size of the contacts that are currently being viewed).<br>
       Expected: Similar to previous.
-      
+
+### Deleting multiple contacts
+
+1. Deleting multiple contact while all contacts are being shown
+    1. Prerequisites: List all contacts using the `list` command. At least 10 contacts in the list.
+    
+    1. Test case: `deletem 8 - 10`<br>
+       Expected: Eighth, ninth and tenth contact deleted from the list. If deleted contact has birthday, it's birthday reminder message is deleted from the birthday reminder list. Details of the deleted contact shown in the status message.
+
+    1. Test case: `deletem 6 - 4`<br>
+       Expected: No contact is deleted since end index smaller than start index. Error details shown in the status message. Status bar remains the same.
+
+    1. Other incorrect delete commands to try: `deletem`, `deletem -2 - -1 `, `deletem x - y`, `...` (where x is not less than y, either x or y is not a positive integer within list size currently being viewed).<br>
+       Expected: Similar to previous.
+
+1. Deleting a contact while some contacts are being shown
+    1. Prerequisites: Viewing only some contacts using the `find` or `findAny` command. Multiple contacts in the list.
```

Comment:
Might be good to indicate there are at least 10 contacts also since you are using deletem 8-10 :)

Reply:
Nice catch thanks!

Reply:
status bar?